apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/utxo_test.exs

```diff
@@ -57,14 +57,13 @@ defmodule OMG.WatcherRPC.Web.Controller.UtxoTest do
       {:put, :utxo,
        {{1000, 0, 0},
         %{amount: 100, creating_txhash: OMG.State.Transaction.raw_txhash(tx), currency: @eth, owner: bob.addr}}},
-      {:put, :block, %{number: 1000, hash: 0, transactions: [tx_encode]}}
+      {:put, :block, %{number: 1000, hash: <<>>, transactions: [tx_encode]}}
     ])
 
     %{
       "utxo_pos" => _utxo_pos,
       "txbytes" => _txbytes,
-      "proof" => proof,
-      "sigs" => _sigs
```

Comment:
What happened to `sigs` - our swagger says it's returned :question: 
https://developer.omisego.co/elixir-omg/docs-ui/?url=master/security_critical_api_specs.yaml#/UTXO/utxo_get_exit_data

I consider it valuable to have 1 test that checks the response structure.

Reply:
the sigs are never used, if I'm not mistaken, submitting an exit doesn't require that. I think this is some ages-old leftovers. Good catch on the swagger though - looks like I have to delete there as well